mangaki/mangaki/management/commands/anidb.py

```diff
@@ -1,53 +1,21 @@
 from django.core.management.base import BaseCommand, CommandError
 from mangaki.utils.anidb import AniDB
-from mangaki.models import Anime, Artist, Role, Staff
+from mangaki.models import Anime, Artist, Role, Staff, ArtistSpelling
 from urllib.parse import urlparse, parse_qs
 import sys
 
-def pick_among(name, contestants):
-    for i, artist in enumerate(contestants):
-        print('%d: %s' % (i, artist))
-    answer = int(input('Who is {}? '.format(name)))
-    if answer < len(contestants):
-        return contestants[answer]
 
 def get_or_create_artist(name):
-    if ' ' in name:
-        parts = name.split()
-        if len(parts) == 2:
-            last, first = parts
-        else:
-            last, first = parts[-1], ' '.join(parts[:-1])  # Diana Wynne Jones
-    else:
-        last, first = name, ''
-    try:
-        contestants = []
-        if Artist.objects.filter(first_name=first, last_name=last).count():
-            contestants = Artist.objects.filter(first_name=first, last_name=last)
-            return Artist.objects.get(first_name=first, last_name=last)
-        elif first != '' and Artist.objects.filter(first_name=first).count():
-            contestants = Artist.objects.filter(first_name=first)
-        elif Artist.objects.filter(last_name=last).count():
-            contestants = Artist.objects.filter(last_name=last)
-    except:
-        pass
-    if contestants:
-        choice = pick_among(name, contestants)
-        if choice:
-            return choice
-    artist = Artist(first_name=first, last_name=last)
-    artist.save()
+    if Artist.objects.filter(name=name).count():
+        return Artist.objects.get(name=name)
```

Comment:
We could probably use `Artist.objects.get_or_create`. BTW it is probably worth it to add an index on the `Artist.name` field.